rules-engine/src/rules_engine/engine.py

```diff
@@ -43,8 +43,10 @@ def average_indoor_temp(
         tstat_setback: temp in F at which the home is set during off hours
         setback_daily_hrs: average # of hours per day the home is at setback temp
     """
-    # again, not sure if we should check for valid values here or whether we can
-    # assume those kinds of checks will be handled at the point of user entry
+    if setback_daily_hrs is None or setback_daily_hrs == 0:
```

Comment:
I feel the `float | None` syntax is a bit awkward. Is there a way to make this parameter optional instead?

Reply:
And if so, note in the function description that this calculates the average temperature given a "normal" temperature and a "setback."